redbot/cogs/audio/core/commands/controller.py

```diff
@@ -655,16 +656,16 @@ async def command_summon(self, ctx: commands.Context):
                 )
             if not self._player_check(ctx):
                 player = await lavalink.connect(
-                    ctx.author.voice.channel,
+                    channel,
                     deafen=await self.config.guild_from_id(ctx.guild.id).auto_deafen(),
                 )
                 player.store("notify_channel", ctx.channel.id)
             else:
                 player = lavalink.get_player(ctx.guild.id)
                 player.store("notify_channel", ctx.channel.id)
                 if (
```

Comment:
```suggestion
               if channel == player.channel and ctx.guild.me in channel.members:
```
Black formatting part 1